src/vcFBX.cpp

```diff
@@ -357,6 +358,9 @@ vdkError vcFBX_Open(vdkConvertCustomItem *pConvertInput, uint32_t everyNth, cons
   pFBX->lastTouchedPoly = 1; // Forces handling 0, then gets set to 0
   pFBX->lastTouchedMesh = 1;
 
+  if (pFBX->pTrivox != nullptr)
```

Comment:
This check is already being done within `vdkTriangleVoxelizer_Destroy`

Reply:
This change is actually unnecessary, as `pFBX->pTrivox` is always `nullptr` at this point (`vcFBX_Close()` destroys it).

Reply:
Oh yeah I got kind of heavy handed when stuff was still leaking after innumerable changes, will do another pass